.werft/util/werft.ts

```diff
@@ -52,13 +64,29 @@ export class Werft {
         console.log(`[${name}|PHASE] ${desc || name}`)
     }
 
+    private sliceBelongsToCurrentPhase(slice) {
+        for (const [name, phase] of Object.entries(this.phases)) {
+            if(phase.slices[slice] && (name != this.currentPhase))
+                throw new Error(`The slice "${slice}" does not belong to the phase "${name}".`);
```

Comment:
The slice does actually belong to `${name}`, doesn't it? I assume the error message wants to say that the slice does not belong to phase `this.currentPhase`.

Reply:
You are right.

Reply:
Fixed: https://github.com/gitpod-io/gitpod/pull/9064/commits/90f90d9cbb9cc243170d0e656bca0cf13e647d70